arches_component_lab/src/arches_component_lab/cards/types.ts

```diff
@@ -1,7 +1,9 @@
-import type { Component } from "vue";
-import type { CardXNodeXWidget } from "@/arches_component_lab/types.ts";
-
-export interface WidgetComponent {
-    component: Component;
-    cardXNodeXWidgetData: CardXNodeXWidget;
+export interface AliasedTileData {
```

Comment:
props.tileData which populates this has a few more values in it besides just "tileid" and "aliased_data".  Should we add those here too for completness?
```
nodegroup
parenttile
provisionaledits
resourceinstance
sortorder
```

Reply:
There are some better types in modular reports, I'm assuming we will want to export them from arches-querysets so we maintain them where the api is maintained?